content/workflow.md

```diff
@@ -0,0 +1,406 @@
+---
+id: workflow
+title: Workflow
+sidebar_label: Workflow
+slug: 
```

Comment:
You need to have a slug. The slug is the URL when the documentation is published.
`slug: /workflow`

Reply:
slug : workflow ? or entire path?